src/Hosting/Server.Abstractions/src/Internal/IHostContextContainer.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Hosting.Server.Abstractions.Internal
```

Comment:
pubternal is gone, change the namespace and move this type to public.

Reply:
Moved